items/models.py

```diff
@@ -12,6 +12,21 @@ class Group(models.Model):
   title = models.CharField(max_length=150, default='Group Tasks')
   view = models.BooleanField(default=True)
   is_deleted = models.BooleanField(default=False)
+  RED = 'red'
+  GREEN = 'green'
+  BLUE = 'blue'
+  ORANGE = 'orange'
+  WHITE = 'white'
+  YELLOW = 'yellow'
+  COLORS = (
+    (RED, 'red'),
```

Comment:
it's confusing. If you have separate variables, you can use them. Like `(RED, RED),` and so on